selenium/che-selenium-test/selenium-tests.sh

```diff
@@ -14,7 +14,24 @@ export CALLER=$(basename $0)
 
 cd $CUR_DIR
 
-mvn clean dependency:unpack-dependencies \
+index=0
+params=($*)
+cheMultiuserValue=${CHE_MULTIUSER:-false}
+for var in "$@"; do
+   if [[ "$var" =~ --multiuser ]]; then
+       cheMultiuserValue=true
+       unset params[$index]                     # remove "--multiuser" from parameters
+       break
+   fi
+   let "index+=1"
+done
+
+export CHE_MULTIUSER=$cheMultiuserValue
+
+set -- "${params[@]}"
```

Comment:
Make it simpler. Just export what you need.

Reply:
We should remove **--multiuser** parameter from list of parameters because of **webdriver.sh** doesn't recognize it and throws an error.

Reply:
That's what I told. In case of multi use assembly:
```
export CHE_MULTIUSER=true
```

And then check environment variable in the code.

Reply:
We should remove _--multiuser_ parameter from parameters because otherwise it will be been passed to **webdriver.sh** as it and will cause an error at start of execution of selenium tests:
> (target/bin/webdriver.sh "$TESTS_SCOPE" **$@**)